HHS/OPRE-OPS
# Ops 441 update user details

## What changed

- Implement `USER_ADMIN` role changing `User` details.
- Deactivate all `UserSession` when changing `UserStatus` to `INACTIVE/LOCKED`

## Issue

https://github.com/HHS/OPRE-OPS/issues/441

## How to test

1. Login as Budget Team Member
2. Navigate to the `User Admin` page.
3. Search for and select User Demo user
4. Update division, roles and set User Demo status to `LOCKED`
5. Log out and Log in with User Demo (Basic) user.
6. Log in should fail and an event should be created in the DB for a failed login.
7. Log in as Budget Team Member
8. Set User Demo status to `ACTIVE`
9. Log out and you should be able to log in with User Demo (Basic) now.

## Screenshots

![image](https://github.com/user-attachments/assets/e6f6ab22-1318-4f40-9d26-0b9e9839ac22)


## Definition of Done Checklist
